apps/aechannel/test/aesc_txs_SUITE.erl

```diff
@@ -2783,7 +2783,7 @@ fp_use_onchain_oracle(Cfg) ->
 
                 % oracle query and answer
                 oracle_query(OQuestion, _ResponseTTL = 100),
-                oracle_response(OResponse, _ResponseTTL = 100),
+                oracle_response(OResponse, _ResponseTTL1 = 100),
```

Comment:
Or
```
oracle_query(OQuestion, ResponseTTL = 100),
oracle_response(OResponse, ResponseTTL),
```

Reply:
.. not sure I like the suggestion .. To do that kind of thing I'd rather move the assign onto its own line.

I suspect we will see more and more code like this with new shiny otp-24 warnings.